Tools/MiniBrowser/Configurations/Base.xcconfig

```diff
@@ -79,3 +79,6 @@ FRAMEWORK_SEARCH_PATHS[sdk=macos*] = $(inherited) $(WK_QUOTED_OVERRIDE_FRAMEWORK
 SDKROOT = macosx.internal;
 
 CODE_SIGN_IDENTITY = -;
+
+WK_PROCESSED_XCENT_FILE=$(TEMP_FILE_DIR)/$(FULL_PRODUCT_NAME).entitlements
```

Comment:
nit: xcconfig style on this line should be:
```suggestion
WK_PROCESSED_XCENT_FILE = $(TEMP_FILE_DIR)/$(FULL_PRODUCT_NAME).entitlements;
```

Also, since it's only used by the MiniBrowser target, would you mind moving this to MiniBrowser.xcconfig? I think some projects put it in Base.xcconfig to avoid duplication, but I don't see why we'd need that generality here.